tensorflow/core/kernels/sparse_fill_empty_rows_op.cc

```diff
@@ -147,7 +151,7 @@ class SparseFillEmptyRowsOp : public OpKernel {
       }
     }
 
-    if (all_rows_full) {
+    if (all_rows_full && rows_are_ordered) {
```

Comment:
this needs a unit test - there are unit tests in python/kernel_tests/

Reply:
Hi, I have added a unit test `testNoEmptyRowsAndUnordered` in python/kernel_tests/sparse_ops_test.py.  Could you please review again? Thanks!